ckanext/switzerland/templates/scheming/form_snippets/relations.html

```diff
@@ -2,19 +2,24 @@
 
 {% set relations_rows = h.ogdch_relations_form_helper(data) %}
 {%- for row in relations_rows -%}
+  <h4 class="relation-label {{row.css_class}}">{{h.scheming_language_text(field.label) ~ ' ' ~ row.index}}</h4>
```

Comment:
I added a header for each relation in the form, just "Relation 1," "Relation 2," etc., because the form looks very cluttered now that there are five text-inputs per relation. :( Not sure how to improve this without spending a lot of extra time on it.

![Screenshot 2025-02-12 at 15-41-42 Datensätze - opendata swiss](https://github.com/user-attachments/assets/81ad7a41-e5bc-4418-9412-605dcac86a44)